example/burrow_integration/permissions.py

```diff
@@ -40,7 +40,7 @@ def create_contract():
     return hex_hash
 
 @integration_helpers.trace
-def grant_permission(adress, permission):
+def grant_permission(address, permission):
```

Comment:
remember also about changing the name here:
`"CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params` (line 57)

Reply:
It was already named `address`. so I think it maybe contains a variable issue here.

Reply:
You are right. I don't know why I didn't notice:D.